pkgs/applications/audio/abcde/default.nix

```diff
@@ -1,16 +1,23 @@
 { lib, stdenv, fetchurl, libcdio-paranoia, cddiscid, wget, which, vorbis-tools, id3v2, eyeD3
 , lame, flac, glyr
 , perlPackages
+, fetchFromGitHub
 , makeWrapper }:
 
-let version = "2.9.3";
+let version = "2.9.3-unstable-2023-04-23";
 in
   stdenv.mkDerivation {
     pname = "abcde";
     inherit version;
-    src = fetchurl {
-      url = "https://abcde.einval.com/download/abcde-${version}.tar.gz";
-      sha256 = "091ip2iwb6b67bhjsj05l0sxyq2whqjycbzqpkfbpm4dlyxx0v04";
+    #src = fetchurl {
+    #  url = "https://abcde.einval.com/download/abcde-${version}.tar.gz";
+    #  sha256 = "091ip2iwb6b67bhjsj05l0sxyq2whqjycbzqpkfbpm4dlyxx0v04";
+    #};
```

Comment:
```suggestion
```
No need to keep this around, we have the git history.
Please also squash all commits, and reword your commit to follow https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions
I'll merge the PR soon after.